Updates to the Cloudflare Zaraz docs (#3798)
cloudflare/cloudflare-docs: content/zaraz/get-started/add-tool.md

# Add a third-party tool

> You can add new third-party tools and load them into your website through the Cloudflare dashboard. If the tool you're adding needs to start actions on other things rather than the default `Pageview` trigger, it is recommeded that you [create the triggers](/zaraz/get-started/create-trigger/) you need first.

Reviewer: ```suggestion
You can add new third-party tools and load them into your website through the Cloudflare dashboard. If the tool you are adding needs to start actions different from the default `Pageview` trigger, it is recommended that you [create the triggers](/zaraz/get-started/create-trigger/) you need first.
```

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login), and select your account and website.
2. Click **Zaraz**.
3. From **Third-party tools**, click **Add new tool**.
4. Choose a tool from the tools library page and click **Continue** to confirm your selection.
5. In **Set up**, configure the settings for your new tool. The information you need to enter will depend on the tool you choose.
6. Click **Save**.

> While your tool is now configured, it needs to have actions defined before it can work. Depending on the tool you chose, Cloudflare Zaraz might automatically create a "Pageview" action. Refer to [Create actions](/zaraz/get-started/create-actions/) to learn how to create additional actions.

Reviewer: ```suggestion
While your tool is now configured, it needs to have actions defined before it can work. Depending on the tool you chose, Cloudflare Zaraz might automatically create a `Pageview` action. Refer to [Create actions](/zaraz/get-started/create-actions/) to learn how to create additional actions.
```

Author: I've been doing `Pageview` for our system trigger, but "Pageview" of our actions, because the first one is hard-coded and set by the system while the others are essentially user-nameable descriptions.

Reviewer: Leaving this as is, per our conversation
